helpers/partner/partners.json

```diff
@@ -3,19 +3,33 @@
     {
       "name": "HPI",
       "url": "https://hpi.de",
-      "logo": "hpi-logo.jpg",
+      "logo": "https://s3.hidrive.strato.com/schul-cloud-hpi/images/partner/hpi-logo.jpg",
       "padding": "20px"
     },
     {
       "name": "MINT-EC",
       "url": "https://www.mint-ec.de/",
-      "logo": "mintec-logo.jpg",
+      "logo": "https://s3.hidrive.strato.com/schul-cloud-hpi/images/partner/mintec-logo.jpg",
       "padding": ""
     },
     {
       "name": "BMBF",
       "url": "https://www.bmbf.de/",
-      "logo": "bmbf-logo.jpg",
+      "logo": "https://s3.hidrive.strato.com/schul-cloud-hpi/images/partner/bmbf-logo.jpg",
```

Comment:
if these files are not loaded directly anymore, please delete them from the project.